tutorials/llm/llama-3/pruning-distillation/README.rst

```diff
@@ -63,17 +73,38 @@ For pruning and distilling the model, you will use the NeMo Framework which is a
 
    jupyter lab --ip 0.0.0.0 --port=8888 --allow-root
 
-4. Then, navigate to `this notebook <./llama3-pruning-distillation-nemofw.ipynb>`_.
+4. Then, navigate to `this notebook <./introduction.ipynb>`_ to get started.
 
+This directory contains a list of notebooks which will go over all the steps to create a distilled 4B model.
+
+:: 
+
+   <$pruning_distillation>
+   └── introduction.ipynb
+   └── 01_data_preparation.ipynb
+   └── 02_teacher_finetuning.ipynb
+   └── 03_a_depth_pruning.ipynb
+   └── 03_b_width_pruning.ipynb
+   └── 04_a_distilling_depth_pruned_student.ipynb
+   └── 04_b_distilling_width_pruned_student.ipynb
+   └── 05_display_results.ipynb
+   
 Results
 ------------------------------------------------------------------------------
-``NOTE:`` This notebook demonstrates the use of the teacher finetuning, pruning and the distillation script. These scripts should ideally be run on a multi-node cluster with a larger ``GLOBAL_BATCH_SIZE`` and ``STEPS`` to see improvement in the validation loss.
+``NOTE:`` This notebook demonstrates the use of the teacher finetuning, pruning and the distillation scripts. These scripts should ideally be run on a multi-node cluster with a larger ``GLOBAL_BATCH_SIZE`` and ``STEPS`` to see improvement in the validation loss.
```

Comment:
fix punctuation

``NOTE:`` This notebook demonstrates the use of the teacher fine-tuning, pruning, and the distillation scripts. These scripts should ideally be run on a multi-node cluster with a larger ``GLOBAL_BATCH_SIZE`` and ``STEPS`` to see improvement in the validation loss.